modules/ccs-gcp-customer-procedure.adoc

```diff
@@ -5,11 +5,17 @@
 // TODO: Same as other module - Better procedure heading that tells you what this is doing
 
 
-The Customer Cloud Subscription (CCS) model allows Red Hat to deploy and manage {product-title} into a customer’s Google Cloud Platform (GCP) project. Red Hat requires several prerequisites in order to provide these services.
+The Customer Cloud Subscription (CCS) model allows Red Hat to deploy and manage {product-title} into a customer's Google Cloud Platform (GCP) project. Red Hat requires several prerequisites to provide these services.
 
 [WARNING]
 ====
-To use {product-title} in your GCP project, the GCP organizational policy constraint, `constraints/iam.allowedPolicyMemberDomains`, cannot be in place.
+To use {product-title} in your GCP project, the following GCP organizational policy constraints cannot be in place:
+
+* `constraints/iam.allowedPolicyMemberDomains`
+* `constraints/storage.uniformBucketLevelAccess`
+* `constraints/compute.restrictLoadBalancerCreationForTypes`
+
+The policy constraint `constraints/compute.vmExternalIpAccess` is unsupported only during installation. You can re-enable the policy constraint after installation.
```

Comment:
Can we include this in the list with maybe a callout or parentheses about only unsupported during installation? I know, from a customer perspective, I just want something that's easy to view when setting these up, and not including the `vmExternalIpAccess` in the list, I worry, would make it too easy to overlook